.coveragerc

```diff
@@ -103,6 +103,7 @@ omit =
     homeassistant/components/notify/free_mobile.py
     homeassistant/components/notify/instapush.py
     homeassistant/components/notify/nma.py
+    homeassistant/components/notify/command_line.py
```

Comment:
I thought that this would be considered depending on external things, can't formulate a meaningful test for this, if you have any good suggestions I will look into it!